src/EnergyPlus/Autosizing/CoolingCapacitySizing.cc

```diff
@@ -456,6 +456,9 @@ Real64 CoolingCapacitySizer::size(EnergyPlusData &state, Real64 _originalValue,
             errorsFound = true;
         }
     }
+    if (this->dataDXCoolsLowSpeedsAutozize) {
+        this->autoSizedValue *= this->dataFractionUsedForSizing;
+    }
```

Comment:
Couldn't you use if (this->dataFractionUsedForSizing > 0.0) here and not need the new global variable?

Reply:
I tried a few other ways to address this issue and finally arrived at this one. I think, I can use your suggestion. Let see the regression results first.

Reply:
The function `BaseSizer::initializeWithinEP()` resets them to 0.0, but there maybe sizing calculation that does not use this function.

Reply:
@rraustad I used `(this->dataFractionUsedForSizing > 0.0)` instead of `this->dataDXCoolsLowSpeedsAutozize` but it resulted in 10 example files and some six unit tests failed due to change in sizing results. I'm going to use the new variable instead.

Reply:
If you set the correct air flow (DataFlowUsedForSizing?) in the new coil model, then call the CapacitySizer, don't you get the correct capacity?

Reply:
I have tried that but did not work for me. The problem is that you need the coil full air flow rate for each coil speed sizing but you can only get the full flow rate only once. As the result you can apply the method for the first speed only and you run to problem for the higher speeds because the DataFlowUsedForSizing will be reset to zero.